nengo/utils/tests/test_testing.py

```diff
@@ -59,6 +55,31 @@ def test_analytics_norecord():
         analytics.get_filepath(ext='npz')
 
 
+def test_logger_record():
+    logger_obj = Logger('nengo.simulator.logs',
+                        'nengo.utils.tests.test_testing',
+                        'test_logger_record')
```

Comment:
And also have it call `logger_obj.__enter__()` and make a callback for exit, à la `plt`?